prelude.dx

```diff
@@ -419,8 +419,7 @@ def vdot (x:n=>Float) (y:n=>Float) : Float = fsum \i. x.i * y.i
 
 -- matmul. Better symbol to use? `@`?
 (**) : (l=>m=>Float) -> (m=>n=>Float) -> (l=>n=>Float) = \x y.
-  y' = transpose y
-  for i k. fsum \j. x.i.j * y'.k.j
+  for i k. fsum \j. x.i.j * y.j.k
```

Comment:
BTW another potential improvement is replacing `fsum \j.` with `sum for j.`. They should be equally fast given that we've added some optimizations.

Reply:
should we remove fusm everywhere and remove that function?
For now i will do so just from transpose